DEV-214

codekoya opened the pull request:
Added changes for thank you modal window.

codekoya committed:
Added changes for Thank you modal window

mateahoward committed:
Update styles.css

mateahoward commented:
Hey @codekoya 👋 This is some fantastic work!! I've never done the click-outside with vanilla JS or the blurry background so I'm very much impressed 😄 It all works very nicely as well! 

I've pushed some styling changes in but I'll tell you here what they are and why 😄 


All I've done is increased the height and width of the modal so it's a bit bigger, I changed its position from 'absolute' to 'fixed' so that when a user tries to scroll the page the modal scrolls with them until they close it, I've added a very thick purple border around it because our website is quiiiite light-themed and some users it might be difficult to spot we have a modal open, then because everything looked way too purple after I added that thick border I coloured our modal button to green - and that's about it 😄 

Great work my lovely, I'm very much impressed and will for sure steal some of these ideas for other projects of ours 😋 I'm gonna approve but will leave @Denice-S to merge in if she's happy with it 💯

codekoya commented:
> Hey @codekoya 👋 This is some fantastic work!! I've never done the click-outside with vanilla JS or the blurry background so I'm very much impressed 😄 It all works very nicely as well!
> 
> I've pushed some styling changes in but I'll tell you here what they are and why 😄
> 
> All I've done is increased the height and width of the modal so it's a bit bigger, I changed its position from 'absolute' to 'fixed' so that when a user tries to scroll the page the modal scrolls with them until they close it, I've added a very thick purple border around it because our website is quiiiite light-themed and some users it might be difficult to spot we have a modal open, then because everything looked way too purple after I added that thick border I coloured our modal button to green - and that's about it 😄
> 
> Great work my lovely, I'm very much impressed and will for sure steal some of these ideas for other projects of ours 😋 I'm gonna approve but will leave @Denice-S to merge in if she's happy with it 💯

That's amazing! I'm glad you liked it! 

Those changes sound great, happy that you added them!! 😄

And I hope you do! I love glassmorphism, had to sneak it in some project haha

DeniceLayton commented:
Hi @codekoya , i'm seconding what @mateahoward  has said -  super work this looks fab and i'm merging it in
